Integrations/integration-cisco-meraki.yml

```diff
@@ -102,28 +103,39 @@ script:
         return text;
     }
 
+    var merakiUrlPrefix = 'https://dashboard.meraki.com/api/v0/';
+    var apiKey = params.apiKey;
+
     var sendRequest = function(url, method, body) {
         var requestUrl = merakiUrlPrefix + url;
-        var apiKey = params.apiKey;
         var requestMethod = method || 'GET';
 
-
         var httpParams = {
             Method: requestMethod,
             Headers: {
                 'X-Cisco-Meraki-API-Key': [apiKey],
                 'Content-Type': ['application/json'],
             },
+            body: body
         };
-        if(body) {
-            httpParams.body = body;
-        }
 
         var res = http(
             requestUrl,
-            httpParams
+            httpParams,
+            false,
+            false,
+            true // no redirect
         );
 
+        if (res.StatusCode === 302) {
```

Comment:
the 'no-redirect' means that http-client is not automatically redirects the request - instead it returns 302/3/4

maybe we should change its name if its not clear. anyway I will add better doc on this integration